api/src/services/organizations/organizations.ts

```diff
@@ -74,6 +76,56 @@ export const getOrCreateOrganization = async (orgName, reportingPeriodName) => {
   }
 }
 
+export const downloadTreasuryFile: MutationResolvers['downloadTreasuryFile'] =
```

Comment:
Would it be possible to add some unit tests for these, maybe in a followup?

Reply:
yes! good call - done 👍